component/admin/controllers/icals.php

```diff
@@ -369,7 +372,7 @@ function save($key = null, $urlVar = null){
 			$icsFileid = $icsFile->store();
 			$message = JText::_( 'ICS_FILE_IMPORTED' );
 		}
-		if (JRequest::getCmd("task") != "icals.reloadall")
+		if (JRequest::getCmd("task") !== "icals.reloadall" && $guest !== 1)
```

Comment:
Also shouldn't we do the same thing for the reloadall method in the controller.

I wonder if this guys site has some strange caching going on which is why these messages appear - I've never heard of this before.  Were you able to recreate the problem?